decidim-meetings/app/commands/decidim/meetings/admin/update_registrations.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module Admin
+      # This command is executed when the user updates the meeting registrations.
+      class UpdateRegistrations < Rectify::Command
+        # Initializes a UpdateRegistrations Command.
+        #
+        # form - The form from which to get the data.
+        # meeting - The current instance of the meeting to be updated.
+        def initialize(form, meeting)
+          @form = form
+          @meeting = meeting
+        end
+
+        # Updates the meeting if valid.
+        #
+        # Broadcasts :ok if successful, :invalid otherwise.
+        def call
+          @meeting.with_lock do
+            return broadcast(:invalid) if @form.invalid?
+            update_meeting_registrations
+          end
+
+          broadcast(:ok)
+        end
+
+        private
+
+        def update_meeting_registrations
+          @meeting.registrations_enabled = @form.registrations_enabled
+
+          if @form.registrations_enabled
+            @meeting.available_slots = @form.available_slots
+            @meeting.registration_terms = @form.registration_terms
+          end
+
+          @meeting.save!
```

Comment:
Should we rescue this at `call`?

Reply:
Talked offline. I think we should provide a global strategy for rescuing errors because since the form is already validated if `save!` raises an error it means we have an error in our application and we need to handle this error i.e. tracking it in Sentry or another one.
I think we can provide a `Decidim::ExceptionHandler` in the future but it is beyond the PR's scope.